example/podpvc.yaml

```diff
@@ -11,8 +12,6 @@ spec:
   resources:
     requests:
       storage: 1Gi
-    limits:
```

Comment:
@toshipp @llamerada-jp I found thant `e2e/testdata/pod-pvc-template.yaml` also contains similar lines of code as follows.
```
spec:
  accessModes:
  - ReadWriteOnce
  resources:
    requests:
      storage: {{ .resourceRequest }}
    limits:
      storage: {{ .resourceLimit }}
```
I want your advice on whether we should fix the code above too in this PR, or it is better to fix it in another PR.

Reply:
We have already deprecated using `limits.storage`, but still support the behavior.
This file is used for the test.

Reply:
Therefore, we do not need to touch it now.

Reply:
@toshipp Should it at least be updated to include the desired behavior? Or a new file created to show the desired behavior?

Reply:
@tylerauerbeck 
I think it is better to describe the new annotation style only in the example to prevent someone from using the old limits style. On the other hand, for the testdata, we need to include both styles.

Reply:
@toshipp  Thank you for your comment. I understand.